pipeline/multiply.py

```diff
@@ -76,40 +46,33 @@ def interval_index(filename):
 
         # Get NOAA Atlas files for this duration
         atlas_files         = glob.glob(os.path.join(atlas_path,'ak*{}a_ams.tif'.format(d_noaa)))
-        atlas_files_lower   = glob.glob(os.path.join(atlas_path,'ak*{}al_ams.tif'.format(d_noaa)))
-        atlas_files_upper   = glob.glob(os.path.join(atlas_path,'ak*{}au_ams.tif'.format(d_noaa)))
         # Ana sort by interval
         atlas_files.sort(key=interval_index)
-        atlas_files_lower.sort(key=interval_index)
-        atlas_files_upper.sort(key=interval_index)
 
         for ts in ["{}-{}".format(x[0],x[1]) for x in TIMESLICES]:
             print("  time period: {}".format(ts), flush=True)
 
             # Find the (downscaled) deltas file. (There should only be one)
-            deltas_file = glob.glob(os.path.join(path,'*_{}_*_{}_{}*_warped.nc'.format(data_group,d,ts)))[0]
+            deltas_file = glob.glob(os.path.join(path,'*_{}_*_{}_{}*_warp.nc'.format(data_group,d,ts)))[0]
 
             ds = xr.open_dataset(deltas_file)
 
-            # Iterate through each variable
-            for (var, atlas_files_list) in zip(
-                ['pf', 'pf-upper', 'pf-lower'], 
-                [atlas_files, atlas_files_upper, atlas_files_lower]
-            ):
-                # Iterate through each return interval
-                for i in range(len(ds.interval)):
-                    arr = ds[var][i,...,...].values
-                    with rasterio.open(atlas_files_list[i]) as tmp:
-                        atlas_arr = tmp.read(1).astype(np.float32)
+            # Iterate through each return interval
+            for i in range(len(ds.interval)):
+                arr       = ds['pf'      ][i,...,...].values
+                arr_upper = ds['pf-upper'][i,...,...].values
```

Comment:
Could probably remove this and the following line? I think it might save time on reading, and we don't need to use these conf bounds until the undiff step, right?